libs/SalesforceSDK/src/com/salesforce/androidsdk/config/LoginServerManager.java

```diff
@@ -350,9 +348,9 @@ private void initSharedPrefFile() {
 	    final Editor edit = settings.edit();
 		for (int i = 0; i < numServers; i++) {
 			final LoginServer curServer = servers.get(i);
-			edit.putString(String.format(SERVER_NAME, i), curServer.name);
-		    edit.putString(String.format(SERVER_URL, i), curServer.url);
-		    edit.putBoolean(String.format(IS_CUSTOM, i), curServer.isCustom);
+			edit.putString(String.format(Locale.US, SERVER_NAME, i), curServer.name.trim());
```

Comment:
`trim()` while reading - for rogue custom URLs that were stored with whitespace in previous versions of the app.